conan/external_tools/bare_mingw_test.py

```diff
@@ -55,3 +56,21 @@ def build_mingw_test(self):
                                                   (install + " -o language=1 -o static=False", 1, False, False)]:
                     from conans.test.integration.basic_build_test import build
                     build(self, cmd, static, pure_c, use_cmake=False, lang=lang)
+
+
+class MinGWUsingCMakeAndMake(unittest.TestCase):
+
+    def build_hello_with_make_test(self):
+        if Version(client_version) < Version("1.1.0-dev"):
+            raise nose.SkipTest('Only >= 1.1.0-dev')
+
+        client = TestClient()
+        files = cpp_hello_conan_files("lib", "1.0")
+        client.save(files)
+        make_location = "/usr/bin/make" if not tools.os_info.is_windows else CONAN_STANDALONE_MAKE_PATH
```

Comment:
Shouldn't be ``CONAN_STANDALONE_MAKE_PATH`` the one that is conditionally set in conf to either "/usr/bin/make" or to "C:/mingw-w64/mingw32/bin/mingw32-make.exe" depending on the system?

Reply:
Sure